ui.frontend/src/utils.js

```diff
@@ -177,6 +184,32 @@ class Utils {
             Utils.registerMutationObserver(formContainer, fieldCreator, fieldSelector, fieldClass);
         });
     }
+
+    /**
+     * Register time based auto save
+     * @param {module:FormView~FormContainer} formContainer - The form container.
+     * @param autoSaveProperties
+     */
+    static setupAutoSave(formContainer, autoSaveProperties) {
+        const enableAutoSave = autoSaveProperties['fd:enableAutoSave'];
+        if (enableAutoSave) {
+            const autoSaveStrategyType = autoSaveProperties['fd:autoSaveStrategyType'];
+            const autoSaveInterval = autoSaveProperties['fd:autoSaveInterval'];
+            const formModel = formContainer.getModel();
+            const saveEndPoint = Utils.getContextPath() + '/adobe/forms/af/save/' + formModel.id;
+            if (autoSaveStrategyType === 'time' && autoSaveInterval) {
+                if (Utils.#autoSaveIntervalId === -1) {
```

Comment:
It will not work in case of multiple forms in a single page

Reply:
Please add test case for multiple forms, we already have an existing test case for this, you can improve the same

Reply:
Added test case